docs/src/content/docs/getting-started.mdx

```diff
@@ -85,3 +85,34 @@ If you forget anything important, Starlight will let you know.
 Once you have created and customized your Starlight website, you can deploy it to a web server or hosting platform of your choice including Netlify, Vercel, GitHub Pages and many more.
 
 [Learn about deploying an Astro site in the Astro docs.](https://docs.astro.build/en/guides/deploy/)
+
+## Updating Starlight
+
+Starlight is an Astro integration, and is updated like any `@astrojs/*` integration:
```

Comment:
Maybe like this? Unsure if we want a tip or just a regular paragraph here.

```suggestion
:::tip
Because Starlight is beta software, there will be frequent updates and improvements. Be sure to update Starlight regularly!
:::

Starlight is an Astro integration, and is updated like any `@astrojs/*` integration:
```